buildSrc/src/main/groovy/org/elasticsearch/gradle/BuildPlugin.groovy

```diff
@@ -82,6 +82,7 @@ class BuildPlugin implements Plugin<Project> {
             )
         }
         project.pluginManager.apply('elasticsearch.java')
+        ElasticsearchJavaPlugin.configureJars(project)
```

Comment:
We shouldn't be calling these methods statically; that is what we are trying to move away from. Can you please move the notice/license configuration to a new method in BuildPlugin, and leave the other stuff intact?

Reply:
I can, however, aren't we also moving away from adding code to the Groovy parts of the build ?  (I can move the code to Groovy and keep in the code in idiomatic Java, but isn't it still is going the wrong way ?) 

> We shouldn't be calling these methods statically

In that case, should these methods not be `public static` ?

Reply:
I will be fishing moving this code to java soon. I'll take care of re-javaizing it.

Reply:
OK ... I pulled the groovy version from git history and removed the Java version. https://github.com/elastic/elasticsearch/pull/56975/commits/908ee17737f1a283cf0535d0bfe102fa65144fb4

Reply:
Sorry that isn't what I meant. *Only* the notice/license code should be moved. That is the problem you are running into right?

Reply:
pulled only the license and notice parts back to the groovy plugin